CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# v0.1.2
+
+- Added a safety check for enforcing that the VersionMaps provided for
+    (de)serialization are up-to-date with the latest versions of the structs.
```

Comment:
It seems to be an alignment issue here. Also, this entry is not accurate, aren't we enforcing this for unions too (and for enums only when serializing)?